### PR TITLE
Give normal warm donkpocket some omnizine

### PR DESCRIPTION
**Pull request by FluffiestFloof**
## About the PR

I always found weird that the normal donk pocket didn't 'generate' omnizine when warmed up compared to the others.

If it's intended, it's a bit weird as it's right under a comment that says they should get +5 nutriment and some omnizine when warm.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/donkpocket.yml [hunk 1]
```diff
@@ -58,7 +58,9 @@
         maxVol: 10
         reagents:
         - ReagentId: Nutriment
-          Quantity: 7
+          Quantity: 8
```
Comment (ficcialfaint) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/donkpocket.yml:61]: Can you please explain why you change that?

**Review by ficcialfaint — APPROVED** [review]
I don't know whether it is intended or not, but the change makes sense. I assume it was just forgotten when added since the original PR adding that was self-merged

File: Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/donkpocket.yml (hunk 1 above)
Comment (Partmedia) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/Baked/donkpocket.yml:61]: This is not explained in the PR, but the comment on line 27 says warm should be +5 nutriment compared to the base. The base for this donk pocket is 3, given on line 17.

**Comment by Partmedia** [comment]
Thank you for your contribution.

**Pull request merged**